common/src/main/scala/explore/components/ConnectionsStatus.scala

```diff
@@ -52,7 +52,7 @@ object ConnectionsStatus {
     .render(_ =>
       AppCtx.withCtx { ctx =>
         <.span(
-          ctx.clients.ExploreDBConnectionStatus(renderStatus("Hasura DB")).when(false),
+          // ctx.clients.ExploreDBConnectionStatus(renderStatus("Hasura DB")).when(false),
```

Comment:
References to Explore DB are left commented out on purpose. In the future we are planning to have a DB for user settings, etc.